src/app/tool-editor/tool-info/tool-info.component.ts

```diff
@@ -0,0 +1,167 @@
+
+import {
+    Component,
+    Input,
+    OnChanges,
+    SimpleChanges
+} from "@angular/core";
+import {WorkflowModel} from "cwlts/models";
+import {SystemService} from "../../platform-providers/system.service";
+import {SBDraft2CommandLineToolModel} from "cwlts/models/d2sb";
+
+@Component({
+    selector: "ct-tool-info",
+    styleUrls: ["./tool-info.component.scss"],
```

Comment:
You can use the same style file  for tool and workflow info because they're effectively identical. There are currently differences/bugs (namely in padding, size of the revisions panel) which would be avoided.